lib/katello/content/file.rb

```diff
@@ -0,0 +1,4 @@
+Katello::ContentPluginManager.register(::Katello::Repository::FILE_TYPE) do
+  description "File Repository types"
+  allow_creation_by_user false
```

Comment:
OK - I asked in part to avoid having to create a mechanism that is only needed because of this one type if in reality it didn't need to be avoided. Can we file an issue then to open file type up to users?